__tests__/runStylelint.test.js

```diff
@@ -0,0 +1,104 @@
+      expect(runStylelintWithConfigFile).toHaveBeenCalledWith(stylelintFile);
+    });
+
+    test('The analysis ends successfully at a directory containing multiple projects with stylelint configured', () => {
+      const directory = '/my-monorepo-project';
+      const stylelintFileFrontProject1 = `${directory}/front-end1/.stylelintrc.json`;
+      const stylelintFileFrontProject2 = `${directory}/front-end2/.stylelintrc.json`;
+
+      findFilesBy.mockReturnValue([stylelintFileFrontProject1, stylelintFileFrontProject2]);
+      runStylelintWithConfigFile
+        .mockReturnValueOnce({ status: 0, outcomes: stylelintResultWithOneError })
+        .mockReturnValueOnce({ status: 0, outcomes: stylelintResultWithOneError });
+
+      expect(runStylelint(directory)).toStrictEqual({ status: 0, outcomes: stylelintResultWithOneError.concat(stylelintResultWithOneError) });
+      expect(findFilesBy).toHaveBeenCalledWith(directory, '.stylelintrc.json');
+      expect(runStylelintWithConfigFile).toHaveBeenCalledTimes(2);
+      expect(runStylelintWithConfigFile).toHaveBeenNthCalledWith(1, stylelintFileFrontProject1);
+      expect(runStylelintWithConfigFile).toHaveBeenNthCalledWith(2, stylelintFileFrontProject2);
+    });
+
+    test('When the directory has no stylelint to analyse, a success status is returned', () => {
+      const directory = '/my-project';
+
+      findFilesBy.mockReturnValue([]);
+
+      expect(runStylelint(directory)).toStrictEqual({ status: 0, outcomes: [] });
+      expect(findFilesBy).toHaveBeenCalledWith(directory, '.stylelintrc.json');
+      expect(runStylelintWithConfigFile).not.toHaveBeenCalled();
+    });
+  });
+
+  describe('Error status', () => {
+    test('The analysis fails at a directory containing only one project with stylelint configured', () => {
+      const directory = '/my-project';
+      const stylelintFile = `${directory}/.stylelintrc.json`;
+
+      findFilesBy.mockReturnValue([stylelintFile]);
+      runStylelintWithConfigFile.mockReturnValue({ status: 1, outcomes: stylelintResultWithOneError });
+
+      expect(runStylelint(directory)).toStrictEqual({ status: 1, outcomes: stylelintResultWithOneError });
+      expect(findFilesBy).toHaveBeenCalledWith(directory, '.stylelintrc.json');
+      expect(runStylelintWithConfigFile).toHaveBeenCalledTimes(1);
+      expect(runStylelintWithConfigFile).toHaveBeenCalledWith(stylelintFile);
+    });
+
+    test('When the directory has one project whose stylelint analysis fails, an error status is returned', () => {
+      const directory = '/my-monorepo-project';
+      const stylelintFileFrontProject1 = `${directory}/front-end1/.stylelintrc.json`;
+      const stylelintFileFrontProject2 = `${directory}/front-end2/.stylelintrc.json`;
+
+      findFilesBy.mockReturnValue([stylelintFileFrontProject1, stylelintFileFrontProject2]);
+      runStylelintWithConfigFile
+        .mockReturnValueOnce({ status: 0, outcomes: stylelintResultWithOneError })
```

Comment:
Se o status da análise é `0` para o projeto `front-end1`, você não deveria fazer o require da fixture `fixtures/stylelint-results/oneError.json`, tipo o seguinte?

```javascript
const stylelintResultWithoutError = require('./fixtures/stylelint-results/noError.json');
```
Depois, você precisaria mudar essa linha para:
```suggestion
        .mockReturnValueOnce({ status: 0, outcomes: stylelintResultWithoutError })
```